micro_brewery/data/knowledge_article.xml

```diff
@@ -214,28 +214,18 @@
     </div>
 </div>
 <p> <br/></p>
-<h2>Deposit Management 🍻</h2>
-<hr contenteditable="false"></hr>
+<h2>Deposit Management ♻️</h2><hr/>
 <p>Manage your deposit more precisely and efficiently than ever before.</p>
 <h3>Deposit assignation</h3>
 <ul>
-    <li>Create deposit products for each different type of consigns.</li>
-    <li>Apply DEP XX taxes to those products, creating some if necessary, and assign them a category with "Is a deposit category" set like the Deposit category.</li>
-    <li>Set a deposit product on your product to apply these taxes. This will ensure that you invoice correctly the price of consigns.</li>
+    <li>Create deposit products for each different type of consigns and assign them a category with "Is a deposit category" set like the Deposit category.</li>
+    <li>Set a deposit product on your product to apply deposit taxes. This will ensure that you invoice correctly the price of consigns.</li>
 </ul>
 <h3>Deposit return from customers</h3>
-<ul><li>
-        Collect deposit when you deliver your customer. It is easy. When validating your delivery, you can directly set a "Return" and precisely count the number of each type of deposit your customer gave you. It will automatically be
-        reflected on his invoice or issue a credit note.
-    </li>
-<li>You can also collect deposit directly on your Point of Sale. Select the correct product and encode a negative quantity.</li>
+<ul>
+    <li>Collect deposit when you deliver your customer. It is easy. When validating your delivery, you can directly set a "Return" and precisely count the number of each type of deposit your customer gave you. It will automatically be reflected on his invoice or issue a credit note.</li>
```

Comment:
```suggestion
    <li>Collect deposits when you deliver to your customer. It is easy. When validating your delivery, you can directly set a "Return" and precisely count the number of each type of deposit your customer gave you. It will automatically be reflected on his invoice or issue a credit note.</li>
```